### ISSUE TITLE
Add analyzer that sorts usings alphabethically

### ISSUE THREAD

**Issue by MovGP0** [JosefPihrt/Roslynator#215]
there should be an analyzer that sorts usings alphabethically. 

Optional: usings for System and Microsoft namespaces should come before others.

**Comment by edwinf** [comment]
@JosefPihrt is this something that might make a good community pull request?

**Comment by tiesmaster** [comment]
For your information, Visual Studio _already_ does this, out of the box:

![image](https://user-images.githubusercontent.com/460904/75152593-af901900-5709-11ea-806b-f837974957a4.png)

This will also remove unused usings, but if you don't like that, you can also trigger the sorting as a dedicated action, without the removal (underneath Edit > IntelliSense > Sort Usings).

This will also sort `System` namespaces on top (by default, I think), [controllable by an option](https://docs.microsoft.com/en-us/visualstudio/ide/reference/options-text-editor-csharp-advanced?view=vs-2019#using-directives). This can also be configured using [`.editorconfig` options](https://docs.microsoft.com/en-us/visualstudio/ide/editorconfig-formatting-conventions?view=vs-2019#organize-using-directives), and applied via [Code Cleanup](https://docs.microsoft.com/en-us/visualstudio/ide/code-styles-and-code-cleanup?view=vs-2019#apply-code-styles).

@MovGP0 Is this of use to you? You mentioned specifically an analyzer in this project, so if you're really looking for something like that, then @JosefPihrt is going to be authoritative on that, otherwise these options might be of use to you ;)

**Comment by JosefPihrt** [comment]
I agree that this feature is already available as a part of "code cleanup".

It could be provided as a Roslynator analyzer but it would need to load options from Visual Studio options and .editorconfig options which is not an easy task.
